### ISSUE TITLE
1.14 beta coexpression - discrepancy in sample numbers

### ISSUE THREAD

**Issue by tmazor** [cBioPortal/cbioportal#4355]
The numbers of samples listed in the dropdown is not the same as the number of samples in the tooltip. Why? Which is the correct number?
![image](https://user-images.githubusercontent.com/10508276/41246277-9f5e03c0-6d78-11e8-8d5b-25a48f141fc7.png)

![image](https://user-images.githubusercontent.com/10508276/41246286-a620e484-6d78-11e8-868a-43d4f3459d6c.png)

http://www.cbioportal.org/beta/index.do?cancer_study_id=lgg_tcga&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&case_set_id=lgg_tcga_cnaseq&gene_list=TP53%2520ATRX%2520PTEN&geneset_list=+&tab_index=tab_visualize&Action=Submit&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=lgg_tcga_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=lgg_tcga_gistic

**Comment by tmazor** [comment]
There's also a discrepancy in the total # of genes - the tooltip says 20532 genes but the table says 20053 genes:
![image](https://user-images.githubusercontent.com/10508276/41246999-b10a3362-6d7a-11e8-8db9-e0644de0e0b0.png)

SImilarly for the microarray data, there are more genes in the table (17157) than in the tooltip description (17155):
![image](https://user-images.githubusercontent.com/10508276/41247083-e78ce56a-6d7a-11e8-9814-c492d6b889f5.png)

**Comment by adamabeshouse** [comment]
The # samples issue is bc the number in parens is the number of those profiled samples which are in the query. Not sure how to solve this confusing UI issue. Eg in this query over All Tumors, the numbers match up: http://www.cbioportal.org/beta/index.do?cancer_study_id=lgg_tcga&Z_SCORE_THRESHOLD=2&RPPA_SCORE_THRESHOLD=2&data_priority=0&case_set_id=lgg_tcga_all&gene_list=TP53%2520ATRX%2520PTEN&geneset_list=%20&tab_index=tab_visualize&Action=Submit&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=lgg_tcga_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=lgg_tcga_gistic&show_samples=false&clinicallist=PROFILED_IN_lgg_tcga_mutations%2CPROFILED_IN_lgg_tcga_gistic

The # genes issue seems most likely a data issue, the description of the profile is wrong.

**Comment by jjgao** [comment]
@adamabeshouse maybe change the text in the drop-down to "283 queried samples"?

**Comment by adamabeshouse** [comment]
sure

**Comment by tmazor** [comment]
I still think this is confusing for a user - I just don't think it's particularly clear that the tooltip refers to something slightly different than the data on the page.

As I see it, there are currently 3 problems with the tooltip:
1) The number of samples in the tooltip won't always agree with the number of samples in the dropdown/query
2) The number of genes in the tooltip doesn't always agree with the number of genes in the table
3) The oncotree code is in lowercase which makes it very hard to read

If I understand correctly, the tooltip text already exists as-is somewhere in the database? And therefore in order to address points 2 & 3 above, those changes would need to be made in the database? I've noticed the lower case oncotree code across many studies, so this would need to be a widespread fix.

At this point, I'm not sure what the tooltip adds. I've looked at a few studies and haven't yet come across a situation where the tooltip provides additional necessary information (although there certainly could be, I just haven't found them yet).

So - while we should clean up the dataset descriptions to address points 2 & 3 - what if for this page we just drop the tooltip?
